### ISSUE TITLE
Adjusting VideoQuality (bitrate) while streaming

### ISSUE THREAD

**Issue by Flyview** [fyhertz/libstreaming#103]
Is there any way to do this currently? I would like to lower the bitrate if the device cannot keep up. I have tried to do a setVideoQuality and a .configure on mSession but that does not work while streaming. Have I missed something?

**Comment by patriciocordova** [comment]
Basically you should try to ping the network regularly to see what is the upload rate and adjust your quality accordingly.

**Comment by Flyview** [comment]
Right, my question is how to adjust the video quality while streaming through this library.

**Comment by patriciocordova** [comment]
The library doesn't have that functionality, nor many of them. You have to implement it yourself.

**Comment by findxain** [comment]
@Flyview hi ,,Tell me which quality u want to set i will help u :)

**Comment by sbaar** [comment]
I have modified the RTP Socket to dynamically adjust bitrate when the queue gets too full.

https://gist.github.com/sbaar/c6dc432c7d175225f53b4c7c9119e02a

**Comment by findxain** [comment]
but u need to be specif with bit-rate for example if you are using 
new VideoQuality(320,240,15,560*1000)) ;
then you have to use bitrate (140,580,560 or  700)*1000
same like that if u are using setVideoQuality(new VideoQuality(640,480,15,700*1000))

then
 you can try 700,1000,1500,2500.
Hope u got.
